test/validation.test.js

```diff
@@ -52,6 +55,30 @@ describe('Schema validation', () => {
     });
   });
 
+  it('Response headers and body matches the schema when charset is specified', async () => {
```

Comment:
@kibertoad, i'm not sure that we need test case for this case as it's covered in api-schema-builder already. i don't want to end up in testing it's functionality twice.

do you know think it's really necessary?

Reply:
I can remove, if you want, basically this is testing whether library itself is supporting this functionality, regardless of what underlying low-level libraries are helping to achieve this purpose.

Reply:
i understand your motivation, but in that case we'll have all our tests duplicated in both projects, so I prefer the tests of this package to focus on its own functionality, hence, running the validations correctly, accurate reports and so on

Reply:
But still, if you think they are really important, we can merge them

Reply:
Nah, I'm good. Removed.